calico-cloud_versioned_docs/version-3.17/image-assurance/scanners/pipeline-scanner.mdx

```diff
@@ -1,73 +1,75 @@
 ---
-description: Scan images using CLI scanner.
+description: Scan images in your build pipeline using Image Assurance.
 ---
 
-# Scan images using the CLI scanner
+# Integrate Image Assurance scanner in your build pipeline
 
 ## Big picture
 
-Scan images manually or via CI/CD pipeline for vulnerabilities using {{prodname}} Image Assurance CLI scanner.
+Integrate the Image Assurance scanner into your build pipeline and private registries to ensure builds are checked by Image Assurance before deployment. You can also use the scanner offline and on-demand for ad hoc scanning and emergency patching. For a real-world use case, see [Hands-on guide: How to scan and block container images to mitigate SBOM attacks](https://www.tigera.io/blog/hands-on-guide-how-to-scan-and-block-container-images-to-mitigate-sbom-attacks/).
+
+The Image Assurance scanner is available in a containerized version so you can easily update it as part of a container-or Kubernetes-based pipeline. For details, see [Image Assurance containerized scanner](https://quay.io/repository/tigera/image-assurance-scanner-cli). This release supports **v1.8.4**.
```

Comment:
@bmckercher123 the main purpose of having a containerized version is not that it's easier to update. It's basically the same level of effort to update either the binary or containerized version. 

The main purpose is that some customers need a containerized version in order to integrate it into their CI/CD pipeline. There might be perhaps some other custom use cases where they prefer a containerized version instead of the binary. But the CI/CD pipeline integration is the reason we did this. 

For example for Crown Castle, they use a CI/CD platform that requires steps in their pipeline be deployable as containers.

Reply:
Out of curiosity, is there any reason we wouldn't want to provide a code snippet here that users can copy and paste to docker pull the image?

```
docker pull quay.io/tigera/image-assurance-scanner-cli:v1.8.4
```

That would be more consistent with the [instructions we already have](https://docs.tigera.io/calico-cloud/image-assurance/scanners/cli-based-scanner#start-the-cli-scanner) for the CLI scanner.